### ISSUE TITLE
Add function to check whether support access is enabled for a device/application

### ISSUE THREAD

**Issue by pimterry** [resin-io/resin-sdk#441]
Also planned for the Python SDK: https://github.com/resin-io/resin-sdk-python/issues/54

Not particularly high priority as far as I know, but do comment below if this is something you'd find useful.

**Comment by lurch** [comment]
Apologies in advance for the stupid question, but does the resin.io dashboard (as viewed by a customer) display this info?

**Comment by pimterry** [comment]
Yes it does, but by retrieving it upfront, as part of the device/application itself, so it doesn't need a separate method to look it up independently.

**Comment by lurch** [comment]
Another stupid question: Does "retrieving it upfront" mean that it doesn't get updated dynamically if it gets turned on or off? (e.g. by another user in the same Org)

**Comment by pimterry** [comment]
We load device data from scratch when you open various pages, and then refresh it at frequent intervals (_very_ frequent - every few seconds), but we're not pushing dynamic updates to open pages or doing much more finely tuned data querying and updating, as far as I know.
